string pose_name
---
bool success
string error_msg
